web/app/components/inputs/product-select.hbs

```diff
@@ -36,11 +36,11 @@
         @offset={{@offset}}
         @matchAnchorWidth={{@matchAnchorWidth}}
         @secondaryFilterAttribute="abbreviation"
-        class="product-select-dropdown-list w-[300px]"
+        class="product-select-dropdown-list"
         ...attributes
       >
         <:anchor as |dd|>
-          <dd.ToggleSelect class="gap-2">
+          <dd.ToggleSelect class="gap-2" id="product-select">
```

Comment:
This is the `@controlID` value on the `label` (https://github.com/hashicorp-forge/hermes/pull/380/files#diff-ef75bd0581dfe8416249c82f232d502485f659443563d9a553c39a100ef453dbR51)

You can now click the label to open the dropdown.